pkg/services/alerting/notifiers/discord.go

```diff
@@ -192,7 +201,9 @@ func (dn *DiscordNotifier) embedImage(cmd *models.SendWebhookSync, imagePath str
 		return err
 	}
 
-	w.Close()
+	if err := w.Close(); err != nil {
```

Comment:
Any reason for closing it twice? Shouldn't the deferred call on the top be good enough?

Reply:
@bergquist See my comment above.